static/js/hotkey.js

```diff
@@ -357,6 +358,10 @@ function process_hotkey(e) {
         case 'narrow_by_subject':
             return do_narrow_action(narrow.by_subject);
         case  'enter': // Enter: respond to message (unless we need to do something else)
+            if (current_view === "#settings" || current_view === "#administration" ||
+                current_view === "#subscriptions") {
+                return false;
+            }
```

Comment:
There's a section earlier in the file that does this:
```
         if (event_name === 'enter') {
             if (activity.searching()) {
                 activity.blur_search();
                 return true;
             } else if (stream_list.searching()) {
                 stream_list.clear_and_hide_search();
                 return true;
             }
         }
```

I think that this code change most logistically belongs up there, not here.